cluster/gce/gci/configure-helper.sh

```diff
@@ -29,7 +29,7 @@ function setup-os-params {
   # Reset core_pattern. On GCI, the default core_pattern pipes the core dumps to
   # /sbin/crash_reporter which is more restrictive in saving crash dumps. So for
   # now, set a generic core_pattern that users can work with.
-  echo "core.%e.%p.%t" > /proc/sys/kernel/core_pattern
+  echo "/core.%e.%p.%t" > /proc/sys/kernel/core_pattern
```

Comment:
Could you please share more about why the issue is occurring?

If we do decide using an absolute path is the best way to address the given issue, I think it'd be helpful to update the comment to ensure that knowledge is accessible and displayed to the user.

Reply:
Reference http://man7.org/linux/man-pages/man5/core.5.html 
"Since Linux 3.6, if /proc/sys/fs/suid_dumpable is set to 2
       ("suidsafe"), the pattern must be either an absolute pathname
       (starting with a leading '/' character) or a pipe, as defined below"

Reply:
Today I learned :) Thank you!